files/root/grafana_tools/backup_and_sync/backup_grafana.sh

```diff
@@ -6,17 +6,19 @@ pushd $(dirname "$0") > /dev/null
 source ./config
 
 ## Get Grafana backing DB type
-db_type=$(grep "^type = " /etc/grafana/grafana.ini | cut -d' '- f 3)
+db_type=$(grep "^type = " /etc/grafana/grafana.ini | cut -d' ' -f 3)
 
 if [ "${db_type}" == "sqlite3" ]; then
-	systemctl stop grafana-server
+	backup_file_name=grafana_backup_$(date +$Y_%m_%d_%H_%M_%S)
 	data_path=$(grep "data = " /etc/grafana/grafana.ini | cut -d' ' -f 3)
-	scp ${data_path}/grafana.db ${destination_host}:/${destination_path}/
+	systemctl stop grafana-server
+        cp -p ${data_path}/grafana.db /tmp/${backup_file_name}
+	#scp ${data_path}/grafana.db ${destination_host}:/${destination_path}/
 	systemctl start grafana-server
 elif [ "${db_type}" == "mysql" ]; then
 	backup_file_name=grafana_backup_$(date +$Y_%m_%d_%H_%M_%S)
 	mysqldump -u ${db_user} -p${db_password} ${db} > /tmp/${backup_file_name}
-	scp /tmp/${backup_file_name} ${destination_host}:/${destination_path}/
+	#scp /tmp/${backup_file_name} ${destination_host}:/${destination_path}/
```

Comment:
Changing this script to ONLY do a backup, and not the sync. There is already a different script for backup & sync.